example/provider-extensions/garden/cloud-profiles/create-cloud-profiles.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
However, this script doesn't seem to be helpful, is it? Why do I need a script for a simple `kubectl apply` command?